src/channeldef.ts

```diff
@@ -911,6 +911,9 @@ export function defaultType<T extends TypedFieldDef<Field>>(fieldDef: T, channel
     case 'shape':
     case 'strokeDash':
       return 'nominal';
+
+    case 'order':
```

Comment:
Add comment to explain why we need to add this ordinal thing? 

(Why does it matter? Does it solve any issue?)

Reply:
Answered offline: the default was nominal but nominal causes a warning.